### ISSUE TITLE
Filter by time

### ISSUE THREAD

**Issue by rafnov** [Droid-ify/client#732]
It's pretty useless for me to browse through dozens of apps that come from Android mid ages which is ap. 10 years ago. In fact I rarely try and use something older than 2 years, because their latest release times prove (non)active developement and that gives hope for bugs/issues fixes.
What I mean is that I need an option to permanently disable / filter apps older than certain year.
I think that *any* repo client should provide that.
Couldn't find that in Droid-ify, best client found so far.

**Comment by rafnov** [comment]
Nope, I don't always need most recent updates but need limit back in time. Like: do not display older than...

**Comment by rafnov** [comment]
Sorry, accidentally closed as I am noob here. Is it visible opened?

**Comment by Iamlooker** [comment]
Aah ok

**Comment by Iamlooker** [comment]
Yes

**Comment by rafnov** [comment]
Can it be done, what I ask?

**Comment by DJCrashdummy** [comment]
this seems to be a duplicate of #635.
